package.json

```diff
@@ -13,7 +13,7 @@
     "build": "snowpack build --config snowpack.prod.config.js",
     "dev": "snowpack dev --config snowpack.dev.config.js",
     "start": "yarn dev",
-    "test": "echo \"Error: no test specified\" && exit 1",
+    "test": "web-test-runner --coverage",
```

Comment:
Can we add a `.github/workflows/ci.yml` file with something like

```
name: CI

on: [push, pull_request]

jobs:
  build:
    runs-on: ubuntu-latest

    steps:
      - uses: actions/checkout@v2
      - name: Use Node.js ${{ matrix.node-version }}
        uses: actions/setup-node@v1
        with:
          node-version: ${{ matrix.node-version }}
      - run: yarn install --frozen-lockfile
      - run: yarn test
      - run: yarn build
```

Something like that -- goal would just be to make sure tests pass and the build succeeds on every push